prisma/seed.ts

```diff
@@ -10,8 +10,10 @@ const prisma = new PrismaClient();
 
 async function main() {
   data.forEach(async (d: Data) => {
```

Comment:
aqui  te aporto con algo de de codigo para probar el seed en la base de datos

```suggestion
    await prisma.$transaction(
    data.map((d: Data) =>
      prisma.invoice.upsert({
        where: { id: d.id },
        update: {},
        create:{
        // .... 
        }
      })
    )
  );
```

Reply:
Aplicados los cambios sugeridos en commit [a66a9a168e287f2518ae5b2422c529c3607f51f1](https://github.com/nsdonato/back-invoices/pull/11/commits/a66a9a168e287f2518ae5b2422c529c3607f51f1)